src/middlewared/middlewared/plugins/interface/netif_linux/utils.py

```diff
@@ -8,7 +8,8 @@
 
 
 INTERNAL_INTERFACES = [
-    "wg", "lo", "tun", "tap", "docker", "veth", "kube-bridge", "kube-dummy-if", "vnet", "openvpn", "macvtap",
+    "wg", "lo", "tun", "tap", "docker", "veth", "kube-bridge", "kube-dummy-if", "vnet",
```

Comment:
I think we want to remove "wg" on here. As i'm not sure users can actually use the wg interface from from app for host-services at the moment.